x-team/auto-ui
# Add codeblock button to richTextEditor

**Release Type:** *RELEASE_TYPE* 


Fixes https://x-team-internal.atlassian.net/browse/XP-2788

## Description
> The task is to add a second toolbar action button `code block` to the RichTextEditor UI component and have both for different purposes.



## Checklist

**Before submitting a pull request,** please make sure the following is done:



## Steps to Test or Reproduce
1- Run `npm start`
2- Go to this URL: http://localhost:9001/?selectedKind=UI%20Components%2FRichTextEditor&selectedStory=basic%20usage&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel&background=%23fff
3- Confirm that the `Code Block` toolbar is there.



## Impacted Areas in Application
- RichTextEditor component


## Screenshots
![add_codeblock_button](https://user-images.githubusercontent.com/5392202/52225995-2a2d9280-2882-11e9-9ca5-651cec5ac7dc.gif)
